roles/manage-jira/README.md

```diff
@@ -92,7 +92,7 @@ atlassian:
 ```
 
 *Note:
-This role  assumes that three groups "project_lead", "project_team_member" and "project_viewer" have already been created.*
+This role  assumes that three groups "project_lead", "project_team_member" and "project_viewer" have already been created. This can be done using the [manage-atlassian-users](https://github.com/redhat-cop/infra-ansible/tree/master/roles/user-management/manage-atlassian-users) role *
```

Comment:
Let's use relative path for the URL so this works regardless of branch/tag/release, etc.